lib/router/handler-info/unresolved-handler-info-by-object.js

```diff
@@ -25,7 +25,7 @@ var UnresolvedHandlerInfoByObject = subclass(HandlerInfo, {
   serialize: function(_model) {
     var model = _model || this.context,
         names = this.names,
-        serializer = this.serializer || (this.handler && this.handler.serialize);
+        serializer = this.serializer || (this.handler && this.handler.serialize && this.handler.serialize.bind(this.handler));
```

Comment:
@trentmwillis it looks like you added this, was this intentional to decoupling serialize from the `this` re: lazy engines? Or an oversight?